CVESerach- bug (#18088)
demisto/content: Packs/CVESearch/ReleaseNotes/1_0_12.md

#### Integrations
##### CVE Search v2

> - fixed an issue with *cve_command* that didn't handle properly multiple or un-valid CVE's

Reviewer: ```suggestion
- Fixed an issue where the ***cve*** command didn't handle multiple CVEs properly.
```
